forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/rs/api/payment/verifier/DetachedJwsVerifier.java

```diff
@@ -84,10 +86,16 @@ public void verifyDetachedJws(String detachedJws, OBVersion obVersion, HttpServl
             }
             UserDetails currentUser = (UserDetails) ((Authentication) principal).getPrincipal();
             Tpp tpp = tppStoreService.findByClientId(currentUser.getUsername()).get();
+            String softwareStatement = tpp.getSsa();
```

Comment:
Not something to change as part of this PR, but it would be nice if `getSsa()` returned a `SoftwareStatement` object instead of a String. As it stands, I think `getSsaJson()` would be a more descriptive name for this method.